l2geth/rollup/sync_service.go

```diff
@@ -86,24 +83,33 @@ func NewSyncService(ctx context.Context, cfg Config, txpool *core.TxPool, bc *co
 	}
 	// Initialize the rollup client
 	client := NewClient(cfg.RollupClientHttp, chainID)
-	log.Info("Configured rollup client", "url", cfg.RollupClientHttp, "chain-id", chainID.Uint64(), "ctc-deploy-height", cfg.CanonicalTransactionChainDeployHeight)
+	log.Info("Configured rollup client", "url", cfg.RollupClientHttp, "chain-id", chainID.Uint64())
```

Comment:
Should we not keep this log as was before?

Reply:
The `ctc-deploy-height` isn't really relevant to the `RollupClient`. I figured it makes sense to group things contextually, it is not logged inside of `initializeLatestL1`